### PR TITLE
docs: add reference

### PR DESCRIPTION
**Pull request by Rory-Z**

### CODE REVIEW

File: docs/en_US/reference/emqxBroker.md [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+
+  - **nodeName** (string)
+
+    NodeName is a request to schedule this pod onto a specific node. If it is non-empty, the scheduler simply schedules this pod onto that node, assuming that it fits resource requirements.
+
++ **env** ([]EnvVar)
+
+  *Patch strategy: merge on key `name`*
+
+  List of environment variables to set in the container. Cannot be updated.
```
Comment (kiragoo) [inline docs/en_US/reference/emqxBroker.md:149]: List of environment variables to set in the container. The key cannot be modified.

Comment (kiragoo) [inline docs/en_US/reference/emqxBroker.md:149]: List of environment variables to set in the container. Cannot be updated unless restart the container.

File: docs/en_US/reference/emqxBroker.md [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+  - **acl.topics**
+
+    - **acl.topics.filter** ([]string)
+
+    - **acl.topics.equal** ([]string)
+
+
+- **plugins**
+
+  - **plugins.name** (string)
```
Comment (kiragoo) [inline docs/en_US/reference/emqxBroker.md:319]: Maybe attach the link of docs which plugins emqx supported  is better?

File: docs/en_US/reference/emqxBroker.md [hunk 3]
```diff
@@ -0,0 +1,445 @@
...
+  - **extraVolumeMounts.subPath** (string)
+
+    Path within the volume from which the container's volume should be mounted. Defaults to "" (volume's root).
+
+  - **extraVolumeMounts.subPathExpr** (string)
+
+    Expanded path within the volume from which the container's volume should be mounted. Behaves similarly to SubPath but environment variable references $(VAR_NAME) are expanded using the container's environment. Defaults to "" (volume's root). SubPathExpr and SubPath are mutually exclusive.
+
+## EmqxBrokerTemplate
+
```
Comment (kiragoo) [inline docs/en_US/reference/emqxBroker.md:286]: Maybe attach the link of emqx docs which related with the field  better?

File: docs/en_US/reference/emqxBroker.md [hunk 4]
```diff
@@ -0,0 +1,445 @@
...
+  *Patch strategy: merge on key `name`*
+
+  List of environment variables to set in the container. Cannot be updated.
+
+  *EnvVar represents an environment variable present in a Container.*
+
+  - **env.name** (string), required
+
+    Name of the environment variable. Must be a C_IDENTIFIER.
```
Comment (kiragoo) [inline docs/en_US/reference/emqxBroker.md:155]: We'd better give the rules for how to convert the key such as "log.level" in emqx docs to env which can be parsed correctly in container.

File: docs/en_US/reference/emqxBroker.md (hunk 3 above)
Comment (Rory-Z) [reply docs/en_US/reference/emqxBroker.md:286]: There is no relevant emqx documentation for `EmqxBrokerTemplate`

**Pull request merged**